app/views/amusement_parks/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>Amusement Park Info</h1>
+
+<p>Name: <%= @amusement_park.name %></p>
+<p>Cost: <%= @amusement_park.admission_cost %></p>
+
+<h2>Assigned Mechanics</h2>
+binding.pry
+
+# I need that AR query to grab the unique mechanic names
```

Comment:
Yup, so this would probably be a method you make in the AmusementPark Model. You could then do something like 
```
def all_mechanics_unique
      mechanics.distinct
end
```
and that would give you the objects, but to get the names you could always add a `.pluck(:name)`. Play with this to see what you can get!